include/types.h

```diff
@@ -164,3 +164,10 @@ enum NETPLAN_EMITTER_ERRORS {
 enum NETPLAN_FORMAT_ERRORS {
     NETPLAN_ERROR_FORMAT_INVALID_YAML,
 };
+
+/**
+ * @brief   Flags used to change the parser behavior.
+ */
+enum NETPLAN_PARSER_FLAGS {
+    NETPLAN_PARSER_IGNORE_ERRORS = 1, ///< Ignore parsing errors such as bad YAML files and definitions.
```

Comment:
nitpick: Maybe we should define this as `0x1` (or `0b1` even) to make it more obvious that this is a flag/bit.

Reply:
I've seen `1 << 0` (and then `1 << 1`, etc), which means you don't even need to know your powers of 2 nor count the number of zeros ;).